types/components/Alert.d.ts

```diff
@@ -12,7 +12,7 @@ declare class AlertHeading<
   As extends React.ReactType = 'div'
 > extends BsPrefixComponent<As> {}
 
-interface AlertProps extends React.HTMLProps<Alert> {
+export interface AlertProps extends React.HTMLProps<Alert> {
```

Comment:
i guess we don't want the exported name for the individual modules here to be `Props`?

Reply:
tslint recomments interfaces all start with I -> IAlertProps

if that is preferable

Reply:
@taion , your hunch is right. The general convention in the React community, including the `react` package, is to use the name `ComponentNameProps` instead of just `Props`, to make imports of multiple props types easier.